storage/src/adb/any/variable/mod.rs

```diff
@@ -223,7 +277,9 @@ impl<E: RStorage + Clock + Metrics, K: Array, V: Codec, H: CHasher, T: Translato
         let mut after_last_commit = None;
         // The set of operations that have not yet been committed.
         let mut uncommitted_ops = HashMap::new();
-        let mut oldest_retained_loc_found = false;
+        let mut current_index = self.oldest_retained_loc;
```

Comment:
current_index -> current_loc ? (I try to use the _loc suffix for all location-containing indices)